jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/main/java/org/jbpm/workbench/pr/backend/server/DataSetDefsBootstrap.java

```diff
@@ -61,13 +61,23 @@ protected void registerDataSetDefinitions() {
                             "log.correlationKey, " +
                             "log.externalId, " +
                             "log.processInstanceDescription, " +
-                            "COALESCE(info.lastModificationDate,log.end_date) as lastModificationDate " +
+                            "COALESCE(info.lastModificationDate,log.end_date) as lastModificationDate, " +
+                            "COALESCE(error.errorCount,0) as errorCount " +
```

Comment:
This is probably my personal preference but how about doing a subquery directly, same as done in here: https://github.com/kiegroup/jbpm-wb/blob/a39bd4dc07c6f13b33abcea785dc7785849a0fde/jbpm-wb-human-tasks/jbpm-wb-human-tasks-backend/src/main/java/org/jbpm/workbench/ht/backend/server/DataSetDefsBootstrap.java#L142